src/Controller/AuthController.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace eLife\Journal\Controller;
+
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+use Symfony\Component\HttpKernel\HttpKernelInterface;
+
+final class AuthController extends Controller
+{
+    public function redirectAction() : Response
+    {
+        if (!$this->get('security.authorization_checker')->isGranted('CAN_AUTHENTICATE')) {
+            throw new NotFoundHttpException('Not found');
```

Comment:
Had to proxy the `HWIOAuthBundle:Connect:redirectToService` to add in the feature flag. Would rather avoid if possible.

Reply:
We could still separate these check from the rest with different private methods, so that is is clear the rest of this class is just delegating and this is they key behavior instead